08/homepage/simple-homepage/app/page.tsx

```diff
@@ -0,0 +1,53 @@
+"use client";
+
+import Link from "next/link";
+
+export default function Page() {
+  return (
+    <>
+      <main className="w-screen h-screen relative">
+        <div className="flex justify-center items-center w-full h-full bg-cover bg-center">
+          <div className="pl-20 md:pl-40 pb-56 md:pb-20 flex flex-col gap-5 z-[10] max-w-[750px]">
+            <h1 className="text-[50px] text-white font-semibold">
+              Maximilian Zoladz, currently studying
+              <br></br>
```

Comment:
```suggestion
              <br />
```
für einen Einzeiler kann man das am besten so machen